bench/bench-tbl_summary.R

```diff
@@ -17,3 +18,9 @@ bench::mark(
   check = FALSE,
   min_iterations = 30
 )
+
+# knitr::knit(input = "bench/benchmark2html.Rmd", output = "bench/benchmark2html.html", envir = new.env())
```

Comment:
Will running this here in this script include the most recent benchmarks in the report?

It could be moved the benchmark workflow.  Here's an example from another workflow I have that knits a document from the workflow yaml

```r
    # Render READEME.md using rmarkdown
    - name: Build README
      run: Rscript -e 'rmarkdown::render("README.Rmd", output_format = "md_document")'
```

Reply:
You're right, I've moved the code to the yaml file.

Reply:
Below this section, I added code to commit and push the newly knitted report to the repo FYI